docs/BUILD.md

```diff
@@ -0,0 +1,232 @@
+# Building the App for App Store Submission
+
+First, clone the repo (requires `git` - see [Dependencies](#dependencies)).  In order to ensure reproducibility, you will always build from a specified `TAG_NAME`.
+
+```
+git clone -b TAG_NAME https://github.com/WorldHealthOrganization/app.git
+cd app
+```
+
+## iOS
+
+Adapting instructions from [Flutter's iOS deployment docs](https://flutter.dev/docs/deployment/ios).
+
+These instructions assume that an app record has already been added to the App Store Connect account. If you haven't done so already, you can follow [these instructions](https://help.apple.com/app-store-connect/#/dev2cd126805).
+
+### Install Dependencies
+
+CocoaPods install (requires `CocoaPods` and `Flutter` - see [Dependencies](#dependencies)):
+
+```
+cd $(git rev-parse --show-toplevel)/client/flutter
+flutter pub get
+cd ios
+pod install
+```
+
+### Review Xcode Build Settings
+
+For detailed procedures and descriptions, see [Prepare for app distribution](https://help.apple.com/xcode/mac/current/#/dev91fe7130a).
+
+Open the Xcode Workspace:
+
+```
+open Runner.xcworkspace # run this in the client/flutter/ios directory
+```
+
+In Xcode:
+
+1. Select **Runner** in the Xcode project navigator, then select the **Runner** target in the settings view sidebar.
+1. Select the General tab.
+1. Do not change the **Display Name** from `Runner`
+1. Verify you have chosen the same **Bundle Identifier** as the app you have registered on App Store Connect.
+1. In the **Signing** section, ensure `Automatically manage signing` is selected, and select the team associated with your Apple Developer account. If other ways of signing code are needed, review the [Code Signing Guide](https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html) and [Create, export, and delete signing certificates](https://help.apple.com/xcode/mac/current/#/dev154b28f09)
+
+### Create a Build Archive
+
+Configure the app version and build:
+
+1. Select **Product > Scheme > Runner**.
+1. Select **Product > Destination > Generic iOS Device**.
+1. Select **Runner** in the Xcode project navigator, then select the **Runner** target in the settings view sidebar.
+1. In the Identity section, update the **Version** number to the user-facing version number you wish to publish.
```

Comment:
These don't apply to flutter:
<img width="400" alt="Screen Shot 2020-03-25 at 3 32 41 AM" src="https://user-images.githubusercontent.com/11729521/77527423-4fe07700-6e49-11ea-9c03-d29813e1e823.png">